File: easy/9_area_of_triangle/area_of_triangle_test.go
package main

import "testing"

func TestAreaOfTriangle(t *testing.T) {
	if areaOfTriangle(2, 3) != 3 {
		t.Error("Failed")
	}
	if areaOfTriangle(3, 4) != 6 {
		t.Error("Failed")
	}
	if areaOfTriangle(4, 5) != 10 {
		t.Error("Failed")
	}
	if areaOfTriangle(5, 6) != 15 {
		t.Error("Failed")
	}
}

File: easy/9_area_of_triangle/area_of_triangle_test.py
import unittest
from area_of_triangle import areaOfTriangle

class TestAreaOfTriangle(unittest.TestCase):
    def test_area_of_triangle(self):
        self.assertEqual(areaOfTriangle(2, 3), 3)
        self.assertEqual(areaOfTriangle(3, 4), 6)
        self.assertEqual(areaOfTriangle(4, 5), 10)
        self.assertEqual(areaOfTriangle(5, 6), 15)
